packages/roosterjs-editor-plugins/lib/plugins/TableResize/editors/TableSelector.ts

```diff
@@ -41,12 +47,13 @@ export default function createTableSelector(
     div.id = TABLE_SELECTOR_ID;
     div.style.width = `${TABLE_SELECTOR_LENGTH}px`;
     div.style.height = `${TABLE_SELECTOR_LENGTH}px`;
-    document.body.appendChild(div);
+    table.insertAdjacentElement('beforebegin', div);
```

Comment:
So this is to insert the handle DIV into editor content, is that right?

I can think about 2 issues:
1. If an auto save happens while the resize handle is showing, it will be saved as part of the content
2. Please try when compose is zoomed (in OWA), if the handle position is still right.

Reply:
Maybe it is ok to insert here. But we need to remove it when getContent() is called, by handling `ExtractContentWithDomEvent` event